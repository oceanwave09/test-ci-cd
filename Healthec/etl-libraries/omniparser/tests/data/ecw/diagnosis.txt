apuid|RecordID|PatientId|PatientAccountNo|EncounterId|EncounterDate|ProviderNPI|ICDCode|Name|AssessmentOnsetDate
280807|280807.17502.2552026.768705|17502|17502|2552026|2022-10-16|1386731784|M54.32|Back pain with left-sided sciatica|
280807|280807.17502.2526032.412280|17502|17502|2526032|2022-10-14|1417179151|Z01.21|Encounter for dental examination and cleaning with abnormal findings|
280807|280807.17502.2465032.347778|17502|17502|2465032|2022-10-12|1417179151|F72|Severe intellectual disabilities|
280807|280807.17502.2465032.444508|17502|17502|2465032|2022-10-12|1417179151|Z01.20|Encounter for dental examination and cleaning without abnormal finding|
280807|280807.17502.2465032.444508|17502|17502|2465032|2022-10-12|1417179151|Encounter for dental|Encounter for dental examination and cleaning without abnormal finding|
280807|280807.17502.2490312.347651|17502|17502|2490312|2022-10-11|1558362426|F40.9|Phobic anxiety disorder, unspecified|
280807|280807.17502.2490312.347653|17502|17502|2490312|2022-10-11|1558362426|F41.1|Generalized anxiety disorder|
280807|280807.17502.2490312.347776|17502|17502|2490312|2022-10-11|1558362426|F70|Mild intellectual disabilities|
280807|280807.17502.2490312.415661|17502|17502|2490312|2022-10-11|1558362426|F43.10|Post traumatic stress disorder|
280807|280807.17502.2490312.416248|17502|17502|2490312|2022-10-11|1558362426|F84.5|Asperger's disorder|
280807|280807.17502.2490327.347653|17502|17502|2490327|2022-10-11|1558362426|F41.1|Generalized anxiety disorder|
280807|280807.17502.2490327.347795|17502|17502|2490327|2022-10-11|1558362426|F84.0|Autistic disorder|
280807|280807.17502.2490327.347805|17502|17502|2490327|2022-10-11|1558362426|F90.2|Attention-deficit hyperactivity disorder, combined type|
280807|280807.17502.2490327.415661|17502|17502|2490327|2022-10-11|1558362426|F43.10|Post traumatic stress disorder|
280807|280807.17502.2490342.347625|17502|17502|2490342|2022-10-11|1558362426|F34.1|Dysthymic disorder|
280807|280807.17502.2490342.683593|17502|17502|2490342|2022-10-11|1558362426|F43.23|Persistent adjustment disorder with mixed anxiety and depressed mood|
280807|280807.17502.2503637.343493|17502|17502|2503637|2022-10-11|1558362426|F84.0|Autism|
280807|280807.17502.2503637.413012|17502|17502|2503637|2022-10-11|1558362426|Z79.899|Other long term (current) drug therapy|
280807|280807.17502.2503637.422677|17502|17502|2503637|2022-10-11|1558362426|H54.0|Blind|
280807|280807.17502.2503637.423194|17502|17502|2503637|2022-10-11|1558362426|F25.0|Schizoaffective disorder, bipolar type with good prognostic features and with catatonia|
280807|280807.17502.2520126.347653|17502|17502|2520126|2022-10-11|1558362426|F41.1|Generalized anxiety disorder|
280807|280807.17502.2520126.663257|17502|17502|2520126|2022-10-11|1558362426|F40.01|Panic disorder with agoraphobia|
280807|280807.17502.2532730.347625|17502|17502|2532730|2022-10-11|1558362426|F34.1|Dysthymic disorder|
280807|280807.17502.2532730.347751|17502|17502|2532730|2022-10-11|1558362426|F63.81|Intermittent explosive disorder|
280807|280807.17502.2532730.414608|17502|17502|2532730|2022-10-11|1558362426|F70|Mild intellectual disability|
280807|280807.17502.2532730.414778|17502|17502|2532730|2022-10-11|1558362426|F90.2|Attention deficit hyperactivity disorder (ADHD), combined type|
280807|280807.17502.2503649.423370|17502|17502|2503649|2022-10-11|1558362426|F31.4|Bipolar 1 disorder, depressed, severe|
280807|280807.17502.2490352.415027|17502|17502|2490352|2022-10-11|1558362426|F41.0|Panic disorder without agoraphobia|
280807|280807.17502.2490352.417555|17502|17502|2490352|2022-10-11|1558362426|F32.9|Depression, unspecified depression type|
