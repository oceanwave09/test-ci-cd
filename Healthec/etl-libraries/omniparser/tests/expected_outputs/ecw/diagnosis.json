[
	{
		"apuid": "280807",
		"encounter_date": "2022-10-16",
		"encounter_id": "2552026",
		"icd_code": "M54.32",
		"name": "Back pain with left-sided sciatica",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6b3043ebbd6b257fb07d3ba37c665a47",
		"record_id": "280807.17502.2552026.768705",
		"row_id": "8523bbbf-d84f-4078-87ae-3cf08c43c155"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-14",
		"encounter_id": "2526032",
		"icd_code": "Z01.21",
		"name": "Encounter for dental examination and cleaning with abnormal findings",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"record_id": "280807.17502.2526032.412280",
		"row_id": "b86aedd4-6346-4b3c-bac7-1827bd2649e9"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-12",
		"encounter_id": "2465032",
		"icd_code": "F72",
		"name": "Severe intellectual disabilities",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"record_id": "280807.17502.2465032.347778",
		"row_id": "946b21df-d916-4e10-97fe-da147df2624c"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-12",
		"encounter_id": "2465032",
		"icd_code": "Z01.20",
		"name": "Encounter for dental examination and cleaning without abnormal finding",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"record_id": "280807.17502.2465032.444508",
		"row_id": "674cac6a-d0e4-4fc9-86c3-31b7aa584f1d"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-12",
		"encounter_id": "2465032",
		"icd_code": "Encounter for dental",
		"name": "Encounter for dental examination and cleaning without abnormal finding",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"record_id": "280807.17502.2465032.444508",
		"row_id": "c559e8f9-c6dd-4666-9d1b-00c15d756a1b"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490312",
		"icd_code": "F40.9",
		"name": "Phobic anxiety disorder, unspecified",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490312.347651",
		"row_id": "40d57234-1d22-4db1-a0c9-aa0e9b641d47"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490312",
		"icd_code": "F41.1",
		"name": "Generalized anxiety disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490312.347653",
		"row_id": "59da5b27-c798-4d31-97e9-0910d9961316"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490312",
		"icd_code": "F70",
		"name": "Mild intellectual disabilities",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490312.347776",
		"row_id": "e82b9390-473f-46af-a989-27efb52c8434"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490312",
		"icd_code": "F43.10",
		"name": "Post traumatic stress disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490312.415661",
		"row_id": "8d22232a-d7c2-4e81-9b52-b5faa112cc65"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490312",
		"icd_code": "F84.5",
		"name": "Asperger's disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490312.416248",
		"row_id": "960657fe-269a-41fc-8187-93798ed2dd65"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490327",
		"icd_code": "F41.1",
		"name": "Generalized anxiety disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490327.347653",
		"row_id": "31f7f8ce-6629-4574-8ab4-3a4d0e3bd773"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490327",
		"icd_code": "F84.0",
		"name": "Autistic disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490327.347795",
		"row_id": "051e6465-d5f1-40c1-8c39-0a547004597e"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490327",
		"icd_code": "F90.2",
		"name": "Attention-deficit hyperactivity disorder, combined type",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490327.347805",
		"row_id": "d0fec6ee-cd95-4baf-9ff5-3d4271133389"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490327",
		"icd_code": "F43.10",
		"name": "Post traumatic stress disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490327.415661",
		"row_id": "2d968624-051e-460f-82b1-a4e097e29212"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490342",
		"icd_code": "F34.1",
		"name": "Dysthymic disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490342.347625",
		"row_id": "06fa57d2-d92f-46a8-844a-3ac495b98033"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490342",
		"icd_code": "F43.23",
		"name": "Persistent adjustment disorder with mixed anxiety and depressed mood",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490342.683593",
		"row_id": "7c820813-41a0-4e07-a6b6-56375300603e"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2503637",
		"icd_code": "F84.0",
		"name": "Autism",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2503637.343493",
		"row_id": "94a6950e-5a8d-4a49-94c4-0a219fa4b28f"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2503637",
		"icd_code": "Z79.899",
		"name": "Other long term (current) drug therapy",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2503637.413012",
		"row_id": "fa2fbbcf-387f-4c8e-86a7-783cd0391bec"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2503637",
		"icd_code": "H54.0",
		"name": "Blind",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2503637.422677",
		"row_id": "16841f22-fefa-4600-8ffe-9d4a7272deed"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2503637",
		"icd_code": "F25.0",
		"name": "Schizoaffective disorder, bipolar type with good prognostic features and with catatonia",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2503637.423194",
		"row_id": "0b208bf5-641e-44bb-8ef5-35c1ca59982d"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2520126",
		"icd_code": "F41.1",
		"name": "Generalized anxiety disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2520126.347653",
		"row_id": "40dd95c8-118d-4419-abbc-473614c7323f"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2520126",
		"icd_code": "F40.01",
		"name": "Panic disorder with agoraphobia",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2520126.663257",
		"row_id": "4805a73c-9feb-48cc-94ba-aa7fa13f4a04"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2532730",
		"icd_code": "F34.1",
		"name": "Dysthymic disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2532730.347625",
		"row_id": "7fc280be-9258-4774-8a3c-2ac6aca4c185"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2532730",
		"icd_code": "F63.81",
		"name": "Intermittent explosive disorder",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2532730.347751",
		"row_id": "3e39a56a-81e2-4b3b-839c-2bd7983a1085"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2532730",
		"icd_code": "F70",
		"name": "Mild intellectual disability",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2532730.414608",
		"row_id": "19fcb843-22e2-4eef-92c6-a4431dbf694b"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2532730",
		"icd_code": "F90.2",
		"name": "Attention deficit hyperactivity disorder (ADHD), combined type",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2532730.414778",
		"row_id": "2c10092e-5d4b-48cd-8581-e69b791548d6"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2503649",
		"icd_code": "F31.4",
		"name": "Bipolar 1 disorder, depressed, severe",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2503649.423370",
		"row_id": "f23d169d-9f8b-43fc-9c8f-8628fe206e53"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490352",
		"icd_code": "F41.0",
		"name": "Panic disorder without agoraphobia",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490352.415027",
		"row_id": "c4ffda22-8331-4b38-a722-2484094b91f7"
	},
	{
		"apuid": "280807",
		"encounter_date": "2022-10-11",
		"encounter_id": "2490352",
		"icd_code": "F32.9",
		"name": "Depression, unspecified depression type",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "354384a6b019ee7f528d1182f2f7c4f2",
		"record_id": "280807.17502.2490352.417555",
		"row_id": "4c8ee168-885f-4bd7-8d9b-5b13c14d9e0c"
	}
]
